tensorflow/src/main/scala/com/spotify/featran/tensorflow/ExampleTransformer.scala

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.featran.tensorflow
+
+import com.spotify.featran.{CollectionType, FeatureBuilder, JsonSerializable}
+import com.spotify.featran.transformers._
+import org.tensorflow.example.{Example, Feature}
+import shapeless.datatype.tensorflow.TensorFlowType
+
+import scala.reflect.ClassTag
+
+/**
+ * ExampleTransformer takes a TFExample of non-transformed values output by Featran
+ * and applies a settings file to transform the Features into the required output object.
+ *
+ * The main benefit on this approach is that when using this class there does not have to
+ * be a JVM dependency on the original scala object or the original spec that were used to
+ * create the TFExample and the Settings file.
+ *
+ *
+ */
+object ExampleTransformer {
+  import TensorFlowType._
+
+  def toFeature(name: String, ex: Example): Option[Feature] = {
+    val fm = ex.getFeatures.getFeatureMap
+    if (fm.containsKey(name)) {
+      Some(fm.get(name))
+    } else {
+      None
+    }
+  }
+
+  def getDouble(name: String): Example => Option[Double] =
```

Comment:
@richwhitjr this is probably a nit 🎨 i would probably remove the `get` i.e `ExampleTransformer.double(name)`

Reply:
@richwhitjr I said that previously mostly coz most of featran api doesnt follow the `get` convention